lib/Raven/Client.php

```diff
@@ -885,6 +885,8 @@ public function capture($data, $stack = null, $vars = null)
             unset($data['site']);
         }
 
+        $data['contexts']['runtime'] =  array('version' => PHP_VERSION, 'name' => 'php');
```

Comment:
Shouldn't this be merged with a possible user value in the runtime context?

Also there is an extra space character before `array`.